rust_snuba/src/consumer.rs

```diff
@@ -80,7 +83,26 @@ pub fn consumer_impl(
     let max_batch_size = consumer_config.max_batch_size;
     let max_batch_time = Duration::from_millis(consumer_config.max_batch_time_ms);
 
-    tracing::info!(?consumer_config.storages, "Starting Rust consumer");
+    for storage in &consumer_config.storages {
+        tracing::info!(
+            "Storage: {}, ClickHouse Table Name: {}, Message Processor: {:?}, ClickHouse host: {}, ClickHouse port: {}, ClickHouse HTTP port: {}, ClickHouse database: {}",
+            storage.name,
+            storage.clickhouse_table_name,
+            &storage.message_processor,
+            storage.clickhouse_cluster.host,
+            storage.clickhouse_cluster.port,
+            storage.clickhouse_cluster.http_port,
+            storage.clickhouse_cluster.database,
+        );
+        if debug_mode {
+            tracing::info!(
+                "ClickHouse username: {}, ClickHouse password: {}",
+                storage.clickhouse_cluster.user,
+                storage.clickhouse_cluster.password
+            );
+        }
+        //tracing::info!(storage.message_processor, "Message Processor")
```

Comment:
Is this supposed to be commented?

Reply:
I forgot to remove this line all together